app.py

```diff
@@ -1,41 +1,54 @@
+from _pytest.assertion import pytest_assertrepr_compare
```

Comment:
We should ideally avoid any leaking from tests into our application code - it looks like this isn't used anyway?